pkg/controller/template/render.go

```diff
@@ -295,6 +307,7 @@ func renderTemplate(config RenderConfig, path string, b []byte) ([]byte, error)
 	funcs["onPremPlatformIngressIP"] = onPremPlatformIngressIP
 	funcs["onPremPlatformShortName"] = onPremPlatformShortName
 	funcs["onPremPlatformKeepalivedEnableUnicast"] = onPremPlatformKeepalivedEnableUnicast
+	funcs["noProxySplit"] = noProxySplit // TODO: use this to properly format noProxy values in templates
```

Comment:
this is just for testing it out - ideally, now that we know that trying to render a unit/dropin with more than 2048 bytes per line results in a failure, we should put something in place that takes care of not doing that while rendering

Reply:
after investigating this resulted in https://issues.redhat.com/browse/GRPA-3091